src/main/java/mc/duzo/persona/data/PlayerData.java

```diff
@@ -74,6 +74,14 @@ public Optional<LivingEntity> findTarget(ServerWorld world) {
         return Optional.of((LivingEntity) target);
     }
 
```

Comment:
not really necessary cus of the method in TargetingUtil

Reply:
No wait, it's not in the util 
it's in that very same class
did you just not see it or something?

Reply:
Comment is meant for the method below

Reply:
Ahh i see it uses ServerWorld, but again why do all this instead of just changing one variable to "World"